### DISCUSSION TITLE
Does yt-dlg no longer support Windows 7 ?

### DISCUSSION THREAD

**Discussion by drkhdrkh** [oleksis/youtube-dl-gui#163] [General]
Both version 1.8.5 and version 1.8.4 (all installed with msi files) can not start under Windows 7. The following error messages are displayed:

(1)

api-ms-win-core-path-l1-1-0.dll is missing.

(2)

No error messages generated. FormatMessageW:The parameter is incorrect.

Error loading Python DLL '<path to> python310.dll'. LoadLibrary:PyInstaller:FormatMessageW failed."

The first error above will not occur if that dll is manually put in the installation folder.

But, if the single-file variant is used, version 1.8.4 can still run under Windows 7, while version 1.8.5 can not.

So, does yt-dlg no longer support Windows 7 and, perhaps, Windows 8 ?

The following post describes a similiar problem:

https://github.com/pyinstaller/pyinstaller/issues/6837

**Comment by oleksis** [comment]
> So, does yt-dlg no longer support Windows 7 and, perhaps, Windows 8 ?

`yt-dlg` require Python and dependencies. Asking to Bing:

> The last version of Python that is compatible with Windows 7 and available for 32-bit systems is Python 3.8.10. Versions of Python after 3.8.10, such as Python 3.9 and above, are not supported on Windows 7. If you’re using Windows 7, you should download Python 3.8.10 to ensure compatibility with your operating system. Remember to check for any required updates or service packs for your Windows 7 system to ensure the best performance and security when using Python.

The binaries/DLL for `yt-dlg` use Python 3.10

My advice is to install a version of Python compatible with your version of Windows architecture and install `yt-dlg` from its repository in [PyPI](https://pypi.org/project/yt-dlg/)

**Reply by drkhdrkh** [reply]
Good to know that there is still a way.

Although I am far from being an expert on programming, I think I'll give it a shot.

Thanks for reaching out. Thank you.
